crates/lib/src/rules/convention/CV09.rs

```diff
@@ -0,0 +1,166 @@
+use ahash::{AHashMap, AHashSet};
+
+use crate::core::config::Value;
+use crate::core::rules::base::{Erased, ErasedRule, LintResult, Rule, RuleGroups};
+use crate::core::rules::context::RuleContext;
+use crate::core::rules::crawlers::{Crawler, SegmentSeekerCrawler};
+
+#[derive(Default, Clone, Debug)]
+pub struct RuleCV09 {
+    blocked_words: AHashSet<String>,
+    blocked_regex: Vec<regex::Regex>,
+    match_source: bool,
+}
+
+impl Rule for RuleCV09 {
+    fn load_from_config(&self, config: &AHashMap<String, Value>) -> Result<ErasedRule, String> {
+        let blocked_words = config["blocked_words"]
+            .as_string()
+            .unwrap_or_default()
+            .split(',')
+            .map(|s| s.to_string().to_uppercase())
+            .collect::<AHashSet<_>>();
```

Comment:
```suggestion
        let blocked_words = config["blocked_words"].as_string().map_or(Default::default(), |it| {
            it.split(',').map(|s| s.to_string().to_uppercase()).collect::<AHashSet<_>>()
        });
```

Otherwise, it will create [''], which will lead to false positive results